docs/csharp/language-reference/operators/null-forgiving.md

```diff
@@ -13,6 +13,9 @@ Available in C# 8.0 and later, the unary postfix `!` operator is the null-forgiv
 
 The null-forgiving operator has no effect at run time. It only affects the compiler's static flow analysis by changing the null state of the expression. At run time, expression `x!` evaluates to the result of the underlying expression `x`.
 
+> [!NOTE]
+> In C# 8 the null-forgiving operator interacts with the [null-conditional operators](member-access-operators.md#null-conditional-operators--and-) in an unexpected way. The expression `x?.y!.z` is parsed as `(x?.y)!.z`. Due to this interpretation `z` is evaluated even if `x` is `null`, which may result in a <xref:System.NullReferenceException>.
```

Comment:
I know commas are frequently optional in English. Comparing to the previous note, this note misses commas after "In C# 8" and "Due to this interpretation". It also lost the "For example" part. Not sure if it was intentional.

Reply:
It was not intentional. If you want to fix this in the same PR (along with the updated wording), I'll merge it.